powerspikegg/frontend/src/app/home/searchbar.component.ts

```diff
@@ -1,15 +1,34 @@
 import { Component, OnInit } from '@angular/core';
+import { QuerySenderService } from '../services/query-sender.service';
+import {Router} from "@angular/router";
+import {summonerQuery} from "../models/summonerQuery";
+
 
 @Component({
   selector: 'app-searchbar',
   templateUrl: './searchbar.component.html',
 })
 export class SearchBarComponent implements OnInit {
 
-  constructor() {
+  private summonerQuery: string;
+
+  constructor(private querySender: QuerySenderService,
+              private router: Router) {
   }
 
   ngOnInit() {
   }
 
+  sendRequest() {
+    if (this.summonerQuery) {
+      console.log("[INFO] Summoner Query sent: " + this.summonerQuery);
+      this.router.navigateByUrl('/summoner');
+    }
+  }
+
+  errorHandler(error: any) {
+    let logStr = '[ERROR] [QUERY SENDER SERVICE] ' + error;
```

Comment:
You can avoid defining this variable here.
`console.log('[ERROR] [QUERY SENDER SERVICE] ' + error)`